### DISCUSSION TITLE
How to resolve 'Error: getaddrinfo ENOTFOUND securecdn.oculus.com'?

### DISCUSSION THREAD

**Discussion by AdamAtFloreo** [RageAgainstThePixel/setup-ovr-platform-util#4] [Q&A]
When running setup-ovr-platform-util I occasionally get this error, causing my whole step to fail. Is there anything I can do to prevent this from happening, or is it just an intermittent network problem?

**Comment by StephenHodgson** [comment]
do you have logs you can share?

yeah this seems more like an oculus problem when it tries to auto update the util.

**Comment by StephenHodgson** [comment]
I use this daily in my live prod environment, so making sure I don't run into this would benefit us both!

**Comment by AdamAtFloreo** [comment]
> do you have logs you can share?
> 
> yeah this seems more like an oculus problem when it tries to auto update the util.

Yes, I do.
Terribly sorry I didn't include more than just the error message. I trust this is what you mean.
![Screenshot 2023-11-21 at 8 38 43 AM](https://github.com/RageAgainstThePixel/upload-meta-quest-build/assets/135663479/f7dc044b-9ae7-4c5e-89e5-999f6580d99c)

**Comment by StephenHodgson** [comment]
Yeah that log is not terribly informational is it? 😅

**Comment by StephenHodgson** [comment]
It could be that they changed the URL to download the executable. I'll double check and see if it's still the same.

**Comment by AdamAtFloreo** [comment]
Not sure, because it's also an intermittent issue.

**Comment by StephenHodgson** [comment]
https://github.com/RageAgainstThePixel/setup-ovr-platform-util/blob/cef36a5a76d54cfad1f84039546d75852fa39cdf/src/index.js#L26

**Comment by AdamAtFloreo** [comment]
Has it changed?
If it makes any difference, I am running this on a self-hosted macOS machine.

**Comment by AdamAtFloreo** [comment]
The link defined in index.js for the macOS download looks exactly the same as the direct download link on the Oculus website, except the latter also defines 'access_token' in the query string. I'm wondering if that could be related.

**Comment by StephenHodgson** [comment]
> Has it changed? If it makes any difference, I am running this on a self-hosted macOS machine.

I also have self hosted Mac I can test.
